k8s/cronjob.yml

```diff
@@ -4,7 +4,7 @@ metadata:
   name: eclipsefdn-github-sync
   namespace: foundation-internal-webdev-apps
 spec:
-  schedule: "*/20 * * * *"
+  schedule: "* 1-23/2 * * *"
```

Comment:
Minor syntax nit: why not use "* */2 * * *" ?

Reply:
Actually lets make this "5 */2 * * *" since using "* */2" would still cause it to run multiple times an hour.

Reply:
Good idea! Thanks Matt!